docs/concepts/notifications.md

```diff
@@ -45,6 +46,57 @@ It's essential to utilize `secret_params` exclusively for handling sensitive inf
 
 See {py:class}`~mlrun.common.schemas.notification.NotificationKind`.
 
+## Mail notifications
+To send mail notifications, you need to use existing SMTP server.
+```python
+mail_notification = mlrun.model.Notification(
+    kind="mail",
+    when=["completed", "error", "running"],
+    name="mail-notification",
+    message="",
+    condition="",
+    severity="verbose",
+    params={
+        "start_tls": True,
+        "use_tls": False,
+        "validate_certs": False,
+        "email_addresses": ["hegeg56369@inikale.com"],
+    },
+)
+```
+We use the [aiosmtplib](https://aiosmtplib.readthedocs.io/en/stable/) library to send the mail notifications.
+The `params` field should contain the following fields:
+ - server_host: The SMTP server host.
+ - server_port: The SMTP server port.
+ - sender_address: The sender email address.
+ - username: The username for the SMTP server.
+ - password: The password for the SMTP server.
+ - email_addresses: The list of email addresses to send the mail to.
+ - start_tls: Whether to start the TLS connection.
+ - use_tls: Whether to use TLS.
+ - validate_certs: Whether to validate the certificates.
+
+About start_tls and use_tls you can read more on [aiosmtplib doc](https://aiosmtplib.readthedocs.io/en/stable/encryption.html).
+If one of the above params wasn't set, the default param from the `mlrun-smtp-config` secret will be used.
```

Comment:
This is the first time the `mlrun-smtp-config` secret was mentioned. what is it?
At least add `"(see below)"`.